roles/test_alerts/tasks/test_creating_a_standard_alert_route_in_alert_manager.yml

```diff
@@ -57,14 +62,19 @@
       /usr/bin/curl -k -u "{{ prom_user }}:{{ prom_pass }}" \
         -g https://{{ prom_url }}/api/v1/alerts \
         | grep 'firing' | grep 'Collectd metrics receive rate is zero' | wc -l
-  register: output
+  register: cmd_output
   changed_when: false
-  failed_when: "output.stdout|int == 0"
+  failed_when: "cmd_output.stdout|int == 0"
 
 
-- name: "Wait 4 minutes to make sure all SG pods are back to normal"
+- name: "Show the command output"
+  ansible.builtin.debug:
+    var: cmd_output
+
+
+- name: "Wait 2 minutes to make sure all SG pods are back to normal"
   ansible.builtin.pause:
-    minutes: 4
+    minutes: 2
   changed_when: false
```

Comment:
Is the changed_when necessary? This was added when the linter was complaining about no-changed-when on the shell task